packages/deno/lib.deno.d.ts

```diff
@@ -0,0 +1,10190 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
```

Comment:
l: Can we somehow avoid vendoring this file? ie. by gitignoring it and generating it in a "prepare" script?

Reply:
I would try to download and generate the types in one go inside the "prepare" script.